<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightClass</class>
 <widget class="QDialog" name="RightClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>627</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>权限管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="management.qrc">
    <normaloff>:/Management/Resources/ico.ico</normaloff>:/Management/Resources/ico.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_26">
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>163</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>查看权限</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_33">
          <item>
           <widget class="QRadioButton" name="employeeClass_search_right_RBTN">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>118</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>按员工类别查询：</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="employeeClass_search_right_CB"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QRadioButton" name="all_search_right_RBTN">
            <property name="maximumSize">
             <size>
              <width>119</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>查询所有信息</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="search_search_right_BTN">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>163</height>
        </size>
       </property>
       <property name="title">
        <string>修改权限</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>员工类别：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="employeeClass_modify_right_CB"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_28">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>权限修改：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="right_modify_right_CB">
            <item>
             <property name="text">
              <string>查询销售情况</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>修改交易清单</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>查看特殊操作</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>查询优惠记录</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>员工管理</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>会员管理</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>权限管理</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>库存管理</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>进货管理</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>商品管理</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="yes_modify_right_RBTN">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>是</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="no_modify_right_RBTN">
            <property name="text">
             <string>否</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="modify_modify_right_BTN">
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="rightInfo_right_TBW">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>员工工号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>商品管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>库存管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>查看特殊操作</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>修改交易清单</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>员工管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>会员管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>进货管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>权限管理</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>查询销售情况</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_right_LB">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="management.qrc"/>
 </resources>
 <connections/>
</ui>
